CMakeLists.txt

```diff
@@ -10,39 +10,24 @@ if (APPLE)
     set(CMAKE_NO_SYSTEM_FROM_IMPORTED TRUE)
 endif()
 
-set(SUPPORTED_PLATFORMS "Win32" "UWP" "Android" "Apple")
-if (ANDROID)
+set(SUPPORTED_PLATFORMS "Android" "Apple" "UWP" "Win32")
+if(ANDROID)
     set(BABYLON_NATIVE_PLATFORM "Android" CACHE STRING "Target platform for Babylon Native.")
```

Comment:
It seems now BABYLON_NATIVE_PLATFORM  is not used except for testing SUPPORTED_PLATFORMS. Do we still want to set it?
Just a check if (!APPLE && !WIN32 && ....) for the FATAL_ERROR unrecognized platform would be fine

Reply:
It's used in a bunch of places in child `CMakeLists.txt`. Do we need to use the CACHE anymore?

Reply:
Regarding the use of CACHE, apparently not. Who knew?